### DISCUSSION TITLE
Can't install via VMware get the message no DEFAULT or UI configuration directive found

### DISCUSSION THREAD

**Discussion by galaxybroz** [Security-Onion-Solutions/securityonion#10318] [Unsupported Versions] [answered]
I have been trying to install Security Onion 2.3 on VMware but to no avail. I set it up to Centos 7 and spin up the VM to this message. If anyone could help me I would much appreciate it. 
![centos](https://github.com/Security-Onion-Solutions/securityonion/assets/61514773/fbb16d87-462e-4de0-8060-3a77c100a6d0)
![help](https://github.com/Security-Onion-Solutions/securityonion/assets/61514773/b3bece69-d9fa-44cb-ad0f-2ebdf970ade0)

**Comment by dougburks** [comment]
Did you verify the ISO image?
https://docs.securityonion.net/en/2.3/download.html
https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md

**Comment by galaxybroz** [comment]
It was a bad signature. But I downloaded it straight from GitHub. Where can I find the right one?
![signature](https://github.com/Security-Onion-Solutions/securityonion/assets/61514773/3f18ed88-0a65-4f24-8846-05d2286eb1ab)

**Comment by dougburks** [comment, marked as answer]
Please try downloading again. If you continue to have problems verifying the signature, try downloading from a different computer or a different network.

**Reply by galaxybroz** [reply]
My computer was not downloading the full iso. I went to a different computer and was able to download the full iso.
